common/src/androidTest/kotlin/GeoJsonAdapterTest.kt

```diff
@@ -0,0 +1,30 @@
+import com.cuhacking.atlas.db.geoJsonAdapter
+import io.github.dellisd.spatialk.geojson.Feature
+import io.github.dellisd.spatialk.geojson.dsl.point
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class GeoJsonAdapterTest {
+
+    @Test
+    fun `Check if GeoJsonAdapter encoding and decoding works`() {
+
+        val featureAsObject= Feature(point(-75.0, 45.0))
+        val featureAsJson = "{" +
+                "\"type\":\"Feature\"," +
+                "\"geometry\":" +
+                "{" +
+                "\"type\":\"Point\"," +
+                "\"coordinates\":[-75.0,45.0]}," +
+                "\"properties\":{}" +
+                "}"
```

Comment:
```suggestion
        val featureAsJson = 
            """
            { "type": "Feature",
              "geometry": {
                  "type": "Point",
                  "coordinates": [-75.0, 45.0],
              },
              "properties": {}
             }
             """
```

Reply:
Yeah, that will probably make it easier to read 😬